core/src/test/java/nl/nn/adapterframework/pipes/FilePipeTest.java

```diff
@@ -44,13 +42,9 @@ public static void before() throws Exception {
 
     @Test
     public void doTestSuccess() throws Exception {
```

Comment:
Why were all parameters removed from this test?

Reply:
Because it was non sense. The parameter is defined with a value and a sessionKey. The sessionkey refers to parameter itself. Parameter toString will be something like 'Parameter name=[p1] defaultValue=[null] sessionKey=[key]'. Then this will be converted(an attempt to convert) to Integer since the type is integer. Morevover, the parameter won't be used anywhere.